lib/dotenv-flow.js

```diff
@@ -74,12 +74,14 @@ function config(options = {}) {
     process.env.NODE_ENV = options.default_node_env;
```

Comment:
Also we need to correct logic for the `default_node_env` option as it is now setting only `process.env.NODE_ENV` that should not be used anymore in the code, except once where we use it as a default if the `node_env` option is not provided.

Reply:
ah I did not notice that. Ok, I'll iron that out and add a testing case for it

Reply:
I'm going to call this one resolved as per below conversation